### PR TITLE
Adds puffy coats, sneakers and tropical outfits

### PR DESCRIPTION
**Pull request by Alliostra**
## About The Pull Request

Adds 4 varieties of tropical shirts to the drip vendor.
Adds 3 varieties of puffy coats and sneakers to the loadout.

Sprites and decriptions by Obey

## Why It's Good For The Game

Drip

## Changelog
:cl:
add: 4 varieties of tropical shirts to the drip vendor
add: 3 varieties of puffy coats to the loadout.
add: 3 varieties of sneakers to the loadout.
/:cl:

### CODE REVIEW

File: code/modules/clothing/suits/miscellaneous.dm [hunk 1]
```diff
@@ -476,3 +476,27 @@
 	body_parts_covered = UPPER_TORSO|ARMS
 	style = STYLE_HIGH
 	price_tag = 400
+
+/obj/item/clothing/suit/storage/puffyblue
+	name = "blue puffy coat"
+	desc = "A stylish, shiny, very blue Aster's Guild branded puffer coat."
```
Comment (hyperioo) [inline code/modules/clothing/suits/miscellaneous.dm:482]:
```suggestion
	desc = "A stylish, shiny, very blue Aster\'s Guild branded puffer coat."
```

File: code/modules/clothing/suits/miscellaneous.dm [hunk 2]
```diff
@@ -476,3 +476,27 @@
...
+	icon_state = "puffycoatblue"
+	item_state = "puffycoatblue"
+	body_parts_covered = UPPER_TORSO|LOWER_TORSO|ARMS
+	siemens_coefficient = 0.7
+
+/obj/item/clothing/suit/storage/puffypurple
+	name = "purple puffy coat"
+	desc = "A stylish, shiny, very purple Aster's Guild branded puffer coat."
```
Comment (hyperioo) [inline code/modules/clothing/suits/miscellaneous.dm:490]:
```suggestion
	desc = "A stylish, shiny, very purple Aster\'s Guild branded puffer coat."
```

File: code/modules/clothing/suits/miscellaneous.dm [hunk 3]
```diff
@@ -476,3 +476,27 @@
...
+	icon_state = "puffycoatpurple"
+	item_state = "puffycoatpurple"
+	body_parts_covered = UPPER_TORSO|LOWER_TORSO|ARMS
+	siemens_coefficient = 0.7
+
+/obj/item/clothing/suit/storage/puffyred
+	name = "crimson puffy coat"
+	desc = "A stylish, shiny, crimson Aster's Guild branded puffer coat."
```
Comment (hyperioo) [inline code/modules/clothing/suits/miscellaneous.dm:498]:
```suggestion
	desc = "A stylish, shiny, crimson Aster\'s Guild branded puffer coat."
```

File: code/modules/clothing/under/miscellaneous.dm [hunk 4]
```diff
@@ -382,8 +382,33 @@ obj/item/clothing/under/genericr
 		rad = 0
 	)
 
+/obj/item/clothing/under/tropicalpink
+	name = "pink tropical shirt"
+	desc = "An Aster's Guild branded neosilk tropical shirt. This one makes you feel nostalgic."
```
Comment (hyperioo) [inline code/modules/clothing/under/miscellaneous.dm:387]:
```suggestion
	desc = "An Aster\'s Guild branded neosilk tropical shirt. This one makes you feel nostalgic."
```

File: code/modules/clothing/under/miscellaneous.dm [hunk 5]
```diff
@@ -382,8 +382,33 @@ obj/item/clothing/under/genericr
...
+	icon_state = "3005vintage"
+	item_state = "3005vintage"
+	style = STYLE_HIGH
 
+/obj/item/clothing/under/tropicalblue
+	name = "blue tropical shirt" 
+	desc = "An Aster's Guild branded neosilk tropical shirt. This one makes you feel out of touch."
```
Comment (hyperioo) [inline code/modules/clothing/under/miscellaneous.dm:394]:
```suggestion
	desc = "An Aster\'s Guild branded neosilk tropical shirt. This one makes you feel out of touch."
```

File: code/modules/clothing/under/miscellaneous.dm [hunk 6]
```diff
@@ -382,8 +382,33 @@ obj/item/clothing/under/genericr
...
+	icon_state = "miamivice"
+	item_state = "miamivice"
+	style = STYLE_HIGH
 
+/obj/item/clothing/under/tropicalblack
+	name = "black tropical shirt"
+	desc = "An Aster's Guild branded neosilk tropical shirt. This one makes you feel like an animal."
```
Comment (hyperioo) [inline code/modules/clothing/under/miscellaneous.dm:401]:
```suggestion
	desc = "An Aster\'s Guild branded neosilk tropical shirt. This one makes you feel like an animal."
```

File: code/modules/clothing/under/miscellaneous.dm [hunk 7]
```diff
@@ -382,8 +382,33 @@ obj/item/clothing/under/genericr
...
+	icon_state = "animalstyle"
+	item_state = "animalstyle"
+	style = STYLE_HIGH
+
+/obj/item/clothing/under/tropicalgreen
+	name = "puke-green tropical shirt"
+	desc = "An Aster's Guild branded neosilk tropical shirt. This one makes you look like puke."
```
Comment (hyperioo) [inline code/modules/clothing/under/miscellaneous.dm:408]:
```suggestion
	desc = "An Aster\'s Guild branded neosilk tropical shirt. This one makes you look like puke."
```

**Review by hyperioo — COMMENTED** [review]
And four missing end of lines in the .dms

**Comment by zombiemimic** [comment]
image
I think you deleted a sprite you didn't mean to

**Comment by Alliostra** [comment]
> image I think you deleted a sprite you didn't mean to

Thanks

File: code/modules/client/preference_setup/loadout/lists/footwear.dm [hunk 8]
```diff
@@ -55,3 +55,15 @@
...
+	display_name = "purple sneakers"
+	path = /obj/item/clothing/shoes/sneakerspurple
+
+/datum/gear/shoes/sneakersblue
+	display_name = "blue sneakers"
+	path = /obj/item/clothing/shoes/sneakersblue
+
+/datum/gear/shoes/sneakersred
+	display_name = "red sneakers"
+	path = /obj/item/clothing/shoes/sneakersred
```
Comment (TheShown911) [inline code/modules/client/preference_setup/loadout/lists/footwear.dm:69]: Newline

File: code/modules/client/preference_setup/loadout/lists/suits.dm [hunk 9]
```diff
@@ -108,3 +108,15 @@
...
+	path = /obj/item/clothing/suit/storage/puffypurple
+	cost = 1
+/datum/gear/suit/puffyblue
+	display_name = "blue puffy coat"
+	path = /obj/item/clothing/suit/storage/puffyblue
+	cost = 1
+/datum/gear/suit/puffyred
+	display_name = "crimson puffy coat"
+	path = /obj/item/clothing/suit/storage/puffyred
+	cost = 1
```
Comment (TheShown911) [inline code/modules/client/preference_setup/loadout/lists/suits.dm:122]: Newline

File: code/modules/clothing/shoes/miscellaneous.dm [hunk 10]
```diff
@@ -128,3 +128,21 @@
...
+	name = "blue sneakers"
+	desc = "A stylish, expensive pair of blue sneakers."
+	icon_state = "sneakersblue"
+	item_state = "sneakersblue"
+
+/obj/item/clothing/shoes/sneakersred
+	name = "red sneakers"
+	desc = "A stylish, expensive pair of red sneakers."
+	icon_state = "sneakersred"
+	item_state = "sneakersred"
```
Comment (TheShown911) [inline code/modules/clothing/shoes/miscellaneous.dm:148]: Newline

File: code/modules/clothing/suits/miscellaneous.dm [hunk 11]
```diff
@@ -476,3 +476,27 @@
...
+	body_parts_covered = UPPER_TORSO|LOWER_TORSO|ARMS
+	siemens_coefficient = 0.7
+
+/obj/item/clothing/suit/storage/puffyred
+	name = "crimson puffy coat"
+	desc = "A stylish, shiny, crimson Aster\'s Guild branded puffer coat."
+	icon_state = "puffycoatred"
+	item_state = "puffycoatred"
+	body_parts_covered = UPPER_TORSO|LOWER_TORSO|ARMS
+	siemens_coefficient = 0.7
```
Comment (TheShown911) [inline code/modules/clothing/suits/miscellaneous.dm:502]: Newline

**Review by Firefox13 — APPROVED** [review]

**Pull request merged**